src/Elastic.Apm/Elastic.Apm.csproj

```diff
@@ -21,18 +21,18 @@
     <PackageTags>apm, monitoring, elastic, elasticapm, analytics, sdk</PackageTags>
   </PropertyGroup>
   <ItemGroup>
-    <Folder Include="Report\"/>
```

Comment:
Do you know how you got these changes? I think Visual Studio does not want to add the new empty spaces before the closing `/>` and most of the .NET devs at Elastic uses JetBrains rider which specifically removes those empty spaces. Could we revert this please?

Reply:
Hi i also use rider, not sure how i got that changes in the commit, probably missed them for some reason. will revert, thanks

Reply:
Hmm.. interesting. I'll keep this in mind... maybe we need to add something to `.editorconfig`... I'm not surre, I was unable to repro this; on my machine Rider automatically removed those empty spaces. Anyway, thanks for reverting! I think we are good here.